salt/modules/blockdev.py

```diff
@@ -110,7 +110,7 @@ def dump(device, args=None):
 
 @decorators.which('sync')
 @decorators.which('mkfs')
-def format_(device, fs_type='ext4', inode_size=None, lazy_itable_init=None):
+def format_(device, fs_type='ext4', force=False, inode_size=None, lazy_itable_init=None):
```

Comment:
@abednarik, the new argument must be added at the end of the arg list to minimize API changes.